pkg/server/apis/v1alpha2/descriptors/stage.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2018 caicloud authors. All rights reserved.
```

Comment:
I think we can remove this for the moment since we don't have a consistent format for it across the project.

Reply:
This is required by license for open source project. I think the [tool](https://github.com/caicloud/nirvana/blob/master/hack/license/apache.go) can help use to maintain this.

Reply:
If this is mandatory, we should check it and ensure all files have this, but now, we don't have.  And the format is different.

```
/*
Copyright 2019 caicloud authors. All rights reserved.
*/
```

```
/*
Copyright 2018 caicloud authors. All rights reserved.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```